contracts/Bridge.sol

```diff
@@ -2,11 +2,14 @@ pragma solidity 0.6.4;
 pragma experimental ABIEncoderV2;
 
 import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/utils/Pausable.sol";
 import "./interfaces/IRelayer.sol";
 import "./interfaces/IDepositHandler.sol";
 import "./interfaces/IBridge.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./interfaces/IMinterBurner.sol";
 
-contract Bridge {
+contract Bridge is Ownable, Pausable {
```

Comment:
Just as an FYI, we have a [Bridge interface](https://github.com/ChainSafe/chainbridge-solidity/blob/cba4128e3daba07024bb93e49fc6722096ea7df6/contracts/interfaces/IBridge.sol) that `Bridge.sol` doesn't use